### ISSUE TITLE
Updating algosdk version

### ISSUE THREAD

**Issue by AustP** [paulfears/snapalgo#2]
Currently this snap is using algosdk version 1.14.0, which was released Mar 2022.

I'm attempting to use the `signLogicSig` method with a TEAL contract that uses AVM v8 but because the algosdk version is so old, it doesn't support that version.

Any plans on updating the algosdk version?

**Comment by pbennett** [comment]
issues/updates should probably go here:
https://github.com/algorandfoundation/algo-metamask

**Comment by AustP** [comment]
Oh yeah, I started out on that repo and ended up on this one after browsing through the code.

It looks like issues are closed on that repo for the time being. Once they are opened, I'll copy this over.

**Comment by paulfears** [comment]
Yes the version of algosdk will be updated I’m working on a rewrite of the
whole sdk and will be fixed then

**Comment by paulfears** [comment]
Oh my bad I thought you were referring to the snapalgo-sdk, the version of
algo can be updated but it requires ottersec taking a quick look at the new
version, and then we have to report the new version to metamask
